os_migrate/roles/conversion_host_content/tasks/main.yml

```diff
@@ -6,10 +6,16 @@
           Conversion Host OS is
           {{ ansible_distribution }} {{ ansible_distribution_version }}
 
+    - name: running pre content hook
+      ansible.builtin.shell: "{{ os_migrate_conversion_host_pre_content_hook | default(omit) }}"
```

Comment:
im thinking about if this can be a blockinfile and then we render and execute the content